brave/brave-core
# Removes Autofill service on Android

Resolves https://github.com/brave/brave-browser/issues/41254



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Install Brave.
2. Go to Android OS autofill settings and make sure Brave is not on the list of autofill providers.

![Screenshot_20241001_171840](https://github.com/user-attachments/assets/309055a9-54cf-4c27-965b-4280b428372e)
